examples/conditional-resource/Snakefile

```diff
@@ -0,0 +1,8 @@
+
+rule withPartitionFlag:
+    resources:
+        partitionFlag="--partition=aSpecificPartition",
+    shell: echo "executed on aSpecificPartition"
+ 
+rule withoutPartitionFlag:
+    shell: echo "executed on whichever partition Slurm chose"
```

Comment:
When I ran `snakefmt` on this file, I realized I made a mistake in the example code I provided you in https://github.com/jdblischak/smk-simple-slurm/issues/16#issuecomment-1490781367. Since the Snakefile is a Python script, the entire command has to be quoted. I was thinking of YAML syntax, where it is implicitly assumed to be a string. I fixed this in 815fa68

Reply:
I missed that - I was using it with a different Snakefile. Thanks!